LEAF_Request_Portal/admin/templates/mod_groups.tpl

```diff
@@ -15,7 +15,10 @@
 
     <main class="main-content">
 
-        <h2><a href="/LEAF_Request_Portal/admin" class="leaf-crumb-link">Admin</a><i class="fas fa-caret-right leaf-crumb-caret"></i>User access</h2>
+       <!-- LEAF-1839 -- rbs -- fix issue with url misdirect due to hard code url
```

Comment:
We don't need to include LEAF ticket info or keep old code within the page.

Reply:
k, old habit